config/fact_check.yml

```diff
@@ -1,18 +1,13 @@
 # Configuration for the fact check system
-# This file is overwritten on deployment
 
 # The main address pattern sent out as the From address on fact check emails
 # If we ever want to change this format, we'll need to support some notion of
 # legacy formats, so we still pick up emails to old addresses
-address_format: "factcheck+dev-{id}@alphagov.co.uk"
+address_format: <%=ENV.fetch("FACT_CHECK_ADDRESS_FORMAT", "factcheck+dev-{id}@alphagov.co.uk") %>
 
-# In deployed versions, this contains connection details for the fact check
-# account, eg:
-#
-#     address: imap.example.com
-#     port: 993
-#     user_name: factcheck@example.com
-#     password: correct horse battery staple
-#     enable_ssl: true
-#
-fetcher: {}
+fetcher:
+  address: <%=ENV.fetch("FACT_CHECK_ADDRESS", "imap.gmail.com") %>
```

Comment:
I don't think `FACT_CHECK_ADDRESS` or `FACT_CHECK_PORT` are set anywhere.  Is it worth just setting them to "imap.gmail.com" and "993" so that I don't get confused?

Reply:
No they're not set, it just relies on the fallback for them as a default. We can indeed set them up in hieradata to be more concrete about them. We could even just hard code them it just makes it difficult to experiment with different imap servers. 

I'm frequently unsure whether it's better to set these sort of things just as fallbacks to allow easy overriding or to set them up in hieradata, as there are quite a few things you want to normally use default but allow overriding.